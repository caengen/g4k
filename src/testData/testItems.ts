import { Item } from "../types";

export const testAItems = [
  {
    id: "1",
    name: "A",
    value: 50
  },
  {
    id: "1a",
    name: "A",
    value: 50
  },
  {
    id: "1b",
    name: "A",
    value: 50
  },
];

export const testBItems = [
  {
    id: "2",
    name: "B",
    value: 30
  },
  {
    id: "2a",
    name: "B",
    value: 30
  }
];

export const testCItems = [
  {
    id: "3",
    name: "C",
    value: 20
  },
  {
    id: "3a",
    name: "C",
    value: 20
  },
  {
    id: "3b",
    name: "C",
    value: 20
  },
];

export const testDItems = [
  {
    id: "4",
    name: "D",
    value: 15
  },
  {
    id: "4a",
    name: "D",
    value: 15
  },
  {
    id: "4b",
    name: "D",
    value: 15
  },
];

export const testItems = [
  ...testAItems,
  ...testBItems,
  ...testCItems,
  ...testDItems
];