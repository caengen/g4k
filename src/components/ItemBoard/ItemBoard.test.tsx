import React from "react";
import { shallow } from "enzyme";
import { ItemBoard } from ".";
import { testItems } from "../../testData";

it("renders correctly", () => {
  const wrapper = shallow(<ItemBoard items={testItems} />);
  expect(wrapper).toMatchSnapshot();
});
