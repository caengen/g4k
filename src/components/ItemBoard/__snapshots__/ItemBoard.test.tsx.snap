// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <ItemCard
    item={
      Object {
        "id": "1",
        "name": "A",
        "value": 50,
      }
    }
    key="1"
  />
  <ItemCard
    item={
      Object {
        "id": "1a",
        "name": "A",
        "value": 50,
      }
    }
    key="1a"
  />
  <ItemCard
    item={
      Object {
        "id": "1b",
        "name": "A",
        "value": 50,
      }
    }
    key="1b"
  />
  <ItemCard
    item={
      Object {
        "id": "2",
        "name": "B",
        "value": 30,
      }
    }
    key="2"
  />
  <ItemCard
    item={
      Object {
        "id": "2a",
        "name": "B",
        "value": 30,
      }
    }
    key="2a"
  />
  <ItemCard
    item={
      Object {
        "id": "3",
        "name": "C",
        "value": 20,
      }
    }
    key="3"
  />
  <ItemCard
    item={
      Object {
        "id": "3a",
        "name": "C",
        "value": 20,
      }
    }
    key="3a"
  />
  <ItemCard
    item={
      Object {
        "id": "3b",
        "name": "C",
        "value": 20,
      }
    }
    key="3b"
  />
  <ItemCard
    item={
      Object {
        "id": "4",
        "name": "D",
        "value": 15,
      }
    }
    key="4"
  />
  <ItemCard
    item={
      Object {
        "id": "4a",
        "name": "D",
        "value": 15,
      }
    }
    key="4a"
  />
  <ItemCard
    item={
      Object {
        "id": "4b",
        "name": "D",
        "value": 15,
      }
    }
    key="4b"
  />
</styled.div>
`;
